src/forms/signpost/SignPost.tsx

```diff
@@ -39,6 +40,7 @@ type SignPostProps = {
     onClick?: Function;
     onClose?: Function;
     dataqa?: string;
+    showCustomOpenAt?: boolean;
```

Comment:
`showCustomOpenAt -> customSignpost`? `openAt `is different prop used for different purpose please don't mix this here

Reply:
changed the prop name to 'customSignPost'